packages/module/patternfly-docs/content/extensions/data-view/examples/Toolbar/Toolbar.md

```diff
@@ -89,7 +114,7 @@ The `useDataViewSelection` hook manages the selection state of the data view.
 
 ```
 
-# Filters
+## Filters
 Enables filtering of data records in the data view and displays the applied filter chips.
 
 ### Toolbar usage
```

Comment:
For the next section: 

The data view toolbar can include a set of filters by passing a React node to the `filters` property. You can use the predefined components `<DataViewFilters>`, `<DataViewTextFilter>`, and `<DataViewCheckboxFilter>` to customize and handle filtering directly in the toolbar. The `<DataViewFilters>` component is a wrapper that allows conditional filtering using multiple attributes. If you need just a single filter, you can use `<DataViewTextFilter>`, `<DataViewCheckboxFilter>`, or a different filter component alone. Props of these filter components are listed at the bottom of this page. 

You can either pass a `value` and `onChange` event to every filter separately, or you can pass `values` and `onChange` to the `<DataViewFilters>` wrapper, which make them available to its children. Props directly passed to child filters have a higher priority than the "inherited" ones. 

### Filters state
The `useDataViewFilters` hook manages the filter state of the data view. It allows you to define default filter values, synchronize filter state with URL parameters, and handle filter changes efficiently.

**Initial values:**
- `initialFilters` object with default filter values (if the filter param allows multiple values, pass an array)
- Optional `searchParams` object for managing URL-based filter state
- Optional `setSearchParams` function to update the URL when filters are modified
- 
The `useDataViewFilters` hook works well with the React Router library to support URL-based filtering. Alternatively, you can manage filter state in the URL using `URLSearchParams` and `window.history.pushState` APIs, or other routing libraries. If no URL parameters are provided, the filter state is managed internally.

**Return values:**
- `filters` object representing the current filter values
- `onSetFilters` function to update the filter state
- `clearAllFilters` function to reset all filters to their initial values